### ISSUE TITLE
(too much) Discontinuous first derivative in `ContinuousRK`

### ISSUE THREAD

**Issue by djanloo** [djanloo/quilt#19]
Just after #18 (about continuity of the interpolation), It must be noted that the first derivative is still a bit too jumpy IMO. 

![bug_NCE3](https://github.com/djanloo/quilt/assets/89815653/192d0dca-d912-43f0-bda8-a6a20a113299)

Here I will gather all I found about this issue.

**Comment by djanloo** [comment]
What the hell is this
![bug_NCE4](https://github.com/djanloo/quilt/assets/89815653/339135dc-4695-4508-b7d9-1c05fb968124)

**Comment by djanloo** [comment]
There is something fundamentally wrong here:

![bug_NCE7](https://github.com/djanloo/quilt/assets/89815653/8982d6cb-68bc-4a23-bb8d-68f7a8d8d868)
![bug_NCE6](https://github.com/djanloo/quilt/assets/89815653/430c7107-d321-440e-aac3-5a41b3b2245c)

The interpolation seems to shift brutally in time.

**Comment by djanloo** [comment]
My bad, I was plotting old data
![bug_NCE8](https://github.com/djanloo/quilt/assets/89815653/9a051f4b-7a16-45a8-87ba-c03688a39ae9)
![bug_NCE9](https://github.com/djanloo/quilt/assets/89815653/69be8451-fd12-4d14-874e-d3d5e374eb9d)

Now the overall trend seems to be correct. Noise buildup will be matter of another issue.
